includes/wc-account-functions.php

```diff
@@ -95,11 +95,16 @@ function wc_get_account_menu_items() {
 		'customer-logout' => get_option( 'woocommerce_logout_endpoint', 'customer-logout' ),
 	);
 
+	$edit_address = __( 'Address', 'woocommerce' );
+	if ( get_option( 'woocommerce_ship_to_countries' ) !== 'disabled' ) {
+		$edit_address = __( 'Addresses', 'woocommerce' );
+	}
+
 	$items = array(
 		'dashboard'       => __( 'Dashboard', 'woocommerce' ),
 		'orders'          => __( 'Orders', 'woocommerce' ),
 		'downloads'       => __( 'Downloads', 'woocommerce' ),
-		'edit-address'    => __( 'Addresses', 'woocommerce' ),
+		'edit-address'    => $edit_address,
```

Comment:
WordPress provides functions to handle plurals, you can replace to just:

```php
_n( 'Address', 'Addresses', (int) wc_shipping_enabled(), 'woocommerce' )
```

Reply:
7f5dd5c9975e81ad8e45b6068675eeedb0d4a96a